### ISSUE TITLE
exception on usage

### ISSUE THREAD

**Issue by spartanPAGE** [spatie/npm-install-peers#10]
```
npm-install-peers
module.js:471
    throw err;
    ^

Error: Cannot find module 'npm'
    at Function.Module._resolveFilename (module.js:469:15)
    at Function.Module._load (module.js:417:25)
    at Module.require (module.js:497:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (C:\Users\spartanPAGE\AppData\Roaming\npm\node_modules\npm-install-peers\dist\index.js:5:11)
    at Module._compile (module.js:570:32)
    at Object.Module._extensions..js (module.js:579:10)
    at Module.load (module.js:487:32)
    at tryModuleLoad (module.js:446:12)
    at Function.Module._load (module.js:438:3)
```

**Comment by sebastiandedeyne** [comment]
Which node & npm versions are you on?

**Comment by spartanPAGE** [comment]
Node v6.10.3, npm 3.10.10

**Comment by sebastiandedeyne** [comment]
If you run `npm install npm -g` first, does that help?

**Comment by sschuberth** [comment]
I had the same issue (node 7.10.0 and npm 4.2.0 on Windows 10) and running `npm install npm -g`  solved it (and installed npm 5.1.0 in `%APPDATA\npm\node_modules\npm`).
